code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -310,33 +317,30 @@
 		visible_message("<span class='notice'>[src] begins repairing the floor.</span>")
 		addtimer(CALLBACK(src, PROC_REF(make_bridge_plating), F), 5 SECONDS)
 
-/mob/living/simple_animal/bot/floorbot/proc/make_floor(turf/simulated/floor/F)
-	if(mode != BOT_REPAIRING)
-		return
-
-	F.broken = FALSE
-	F.burnt = FALSE
-	F.ChangeTurf(/turf/simulated/floor/plasteel)
-	mode = BOT_IDLE
-	amount--
-	update_icon(UPDATE_OVERLAYS)
-	anchored = FALSE
-	target = null
+//	/mob/living/simple_animal/bot/floorbot/proc/make_floor(turf/simulated/floor/F)
+//		if(mode != BOT_REPAIRING)
+//			return
+//
+//		F.broken = FALSE
+//		F.burnt = FALSE
+//		F.ChangeTurf(/turf/simulated/floor/plasteel)
+//		mode = BOT_IDLE
+//		amount--
+//		update_icon(UPDATE_OVERLAYS)
+//		anchored = FALSE
+//		target = null
```

Comment:
Forgot this commented here?

Reply:
YUP

So, if you can, can you double check this proc is not called? I couldn't find it, nor perceived any difference in game. Just to make sure

Reply:
Its not being used anywhere, see how and where this exact behavior is being done in the code.

Reply:
Thx for the double check, I gotta remember to remove commented out code before committing
Will update tonight

Reply:
[42632cf](https://github.com/ParadiseSS13/Paradise/pull/23162/commits/42632cf25ee4e0e90a3913651236fe22f29b88c5)